ably.d.ts

```diff
@@ -986,6 +986,17 @@ declare namespace Types {
     modes?: ChannelModes;
   }
 
+
+  /**
+   * Passes an attachment to a {@link ChannelBase} name to give a new derived channel
+   */
+  interface DeriveOptions {
+    /**
+     * The filter string to be used to derive new channel.
```

Comment:
This is public facing documentation so it would probably help to mention here that the filter string should be a JMESPath query.

Also, might be worth adding the new docstrings to https://github.com/ably/sdk-api-reference/blob/main/descriptions.md